class/defaults.yml

```diff
@@ -107,8 +110,8 @@ parameters:
       source:
         opensource: https://github.com/isovalent/olm-for-cilium/archive/main.tar.gz
         enterprise: <CILIUM-ENTERPRISE-OLM-MANIFESTS-TARGZ-URL> # Configure the URL in your global defaults.
-      version: "1.14"
-      patchlevel: "7"
+      version: "1.15"
+      patchlevel: "1"
```

Comment:
Is the OLM version different?

Reply:
yes, in the sense that there's no opensource OLM 1.16 release yet in https://github.com/isovalent/olm-for-cilium/tree/main/manifests